src/pages/index.tsx

```diff
@@ -1,18 +1,11 @@
-import { Text } from "@nextui-org/react";
+// import { Text } from "@nextui-org/react";
+import Hero from "src/components/hero/Hero";
 
 export default function Home() {
   return (
     <div>
```

Comment:
Use implicit [React Fragments](https://reactjs.org/docs/fragments.html#short-syntax) here for optimization concerns.